### PR TITLE
✨ 7 - Testimonials component

### PR DESCRIPTION
**Pull request by bradystroud**
For issue #7 and #21

~~Done video https://www.loom.com/share/3cc35f9f0c9e465381015c3f4eb21afd~~

### CODE REVIEW

**Review by tkapa — DISMISSED** [review]
lgtm

**Review by tkapa — DISMISSED** [review]

File: styles.css [hunk 1]
```diff
@@ -14,4 +14,15 @@
 
 @layer components {
   @import "react-responsive-carousel/lib/styles/carousel.min.css";
-}
\ No newline at end of file
+}
+
+.testimonialBubble:after {
```
[the remark this answers is not in the document]
Comment (wicksipedia) [reply styles.css:19]: how about putting it next to the component as a fallback

File: .tina/queries/queries.gql [hunk 2]
```diff
@@ -10,4 +10,17 @@ query consultingContentQuery($relativePath: String!) {
   consulting(relativePath: $relativePath) {
     ...ConsultingParts
   }
-}
\ No newline at end of file
+}
+
+query allTestimonialsQuery {
+  testimonialsConnection {
+    edges {
```
Comment (wicksipedia) [inline .tina/queries/queries.gql:17]: can we rename the results as this litters the code with edge/node references which is very abstract

**Review by wicksipedia — CHANGES_REQUESTED** [review]

File: .tina/queries/queries.gql (hunk 2 above)
Comment (bradystroud) [reply .tina/queries/queries.gql]: Done - I gave names nodes and edges

**Review by wicksipedia — APPROVED** [review]
lgtm

**Pull request merged**